pmoves/tools/agentgym_field_runner.py

```
@@ -0,0 +1,474 @@
+        return False
+
+
+def _http_post(url: str, payload: dict, timeout: float = 30.0) -> dict:
+    """POST JSON to *url* and return parsed JSON response."""
+    data = json.dumps(payload).encode("utf-8")
+    req = urllib.request.Request(
+        url,
+        data=data,
+        headers={"Content-Type": "application/json"},
+        method="POST",
+    )
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)